Locals/enUS.lua

```diff
@@ -504,12 +562,14 @@ function CraftSim.LOCAL_EN:GetData()
         [CraftSim.CONST.TEXT.CRAFT_QUEUE_CRAFT_BUTTON_ROW_LABEL_WRONG_GEAR] = "Wrong Tools",
         [CraftSim.CONST.TEXT.CRAFT_QUEUE_CRAFT_BUTTON_ROW_LABEL_NO_MATS] = "No Materials",
         [CraftSim.CONST.TEXT.CRAFT_QUEUE_ADD_OPEN_RECIPE_BUTTON_LABEL] = "Add Open Recipe",
+        [CraftSim.CONST.TEXT.CRAFT_QUEUE_ADD_FIRST_CRATFS_BUTTON_LABEL] = "Add First Crafts",

```

Comment:
Typo

Reply:
I didn't find it

Reply:
CRAFT_QUEUE_ADD_FIRST_**CRATFS**_BUTTON_LABEL

Reply:
oops